scripts/Project/ProjectPython.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+from os import path
+from logging import error, info, warning
+from scripts.common import OSSFUZZ, OSSFUZZ_SCRIPTS_HOME
+from .ProjectBase import Project
+from scripts.demangle import main as main_post_process
+
+
+class ProjectPython(Project):
+    def build_w_pass(self, build_script: str = "build_w_pass.sh"):
```

Comment:
@ethanbwang here is a template for working with Python. Ideally, we want to transform `build.sh` or `fuzz_*.py` so that our reporter can be loaded. If this transformation can be done algorithmically (by inserting to text or AST), then we can also implement `auto_build_w_pass` for python.